_includes/procurement-list.html

```diff
@@ -0,0 +1,25 @@
+<div class="row row--gutters column contract-list">
+  <h2 class="CL--title">Open Procurements</h2>
+  {% assign openProcurements = site.procurements | sort: 'position' | where: 'Is Open', true %}
+  {% if openProcurements.size > 0 %}
```

Comment:
I have issues with large conditional blocks in templates because they can make maintenance of the page more difficult further down the road, especially if we don't have prettier support to auto fix things here.  Typically, it's better to toggle between template fragments instead of dropping the markup here if it gets too unbearable.